src/libPMacc/include/cudaSpecs.hpp

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2015 Heiko Burau
+ * Copyright 2015-2017 Heiko Burau
```

Comment:
;) probably an open PR during the update or something pasted in later from an outdated local branch of him.